src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -40,10 +40,17 @@ object Arisa : ConfigSpec() {
 
         object Piracy : ModuleConfigSpec() {
             val piracyMessage by optional(
-                "You are currently using a *non-authorized* version of Minecraft." +
-                        " If you wish to purchase the full game, please visit the [Minecraft Store|https://minecraft.net/store].\r\n" +
-                        "We will not provide support for pirated versions of the game, these versions are modified and may contain malware."
-            )
+                "*Thank you for your report!*\n" +
+                        "However, this issue is {color:#FF5722}*Invalid*{color}.\n" +
+                        "\n" +
+                        "You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].\n" +
+                        "We will not provide support for pirated versions of the game, these versions are modified and may contain malware.\n" +
+                        "\n" +
+                        "\n" +
+                        "*Quick Links*:\n" +
+                        "\uD83D\uDCD3 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- \uD83D\uDCAC [Community Support|https://discord.gg/58Sxm23] -- \uD83D\uDCE7 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- \uD83D\uDCD6 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]"
+                        {color:#bbb}-- "I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}"
```

Comment:
The opening quotation marks of this need to be in fornt of the color, not the I am a bot.

Since this is pasted all of the place, that might actually be the only reason the build fails now.

Reply:
I'll correct that

Reply:
Pushed changes